src/Uno.UI/UI/Xaml/UIElement.wasm.cs

```diff
@@ -433,18 +442,19 @@ public bool Dispatch(EventArgs eventArgs, string nativeEventPayload)
 		internal void RegisterEventHandler(
 			string eventName,
 			Delegate handler,
+			RoutedEvent routedEvent = null,
 			bool onCapturePhase = false,
 			bool canBubbleNatively = false,
 			HtmlEventFilter? eventFilter = null,
 			HtmlEventExtractor? eventExtractor = null,
-			Func<string, EventArgs> payloadConverter = null,
-			Func<EventArgs, bool> eventFilterManaged = null)
```

Comment:
Not used anymore ?

Reply:
Has never been used... never called nor implemented :-)